vmdb/app/models/miq_schedule.rb

```diff
@@ -336,8 +336,8 @@ def depot_hash
   end
 
   def next_interval_time
-    unless self.valid? || self.errors.on(:run_at).empty?
-      $log.warn("Miq(Schedule.next_interval_time) Invalid schedule [#{self.id}] [#{self.name}]: #{self.errors.on(:run_at).to_miq_a.join(", ")}")
+    unless self.valid? || errors[:run_at].nil? || errors[:run_at].empty?
```

Comment:
checking for `nil? || empty?` is the same as checking for `.blank?`.  Can we just simplify it to that?